core/backend/ios/core-test-runner/Config.xcconfig

```diff
@@ -0,0 +1,11 @@
+//
+//  Config.xcconfig
+//  core-test-runner
+//
+//  Created by Daniel Toby on 3/8/22.
```

Comment:
Can you remove all headers that have your name?

Reply:
Looks like some of these are still here.

Reply:
Fixed.